frontend/src/components/OperationStepForm/OperationStepForm.tsx

```diff
@@ -37,6 +37,7 @@ const schema = Yup.object().shape({
 const queries = [
   { key: 'select', icon: 'check', text: 'Select', value: 'select' },
   { key: 'filter', icon: 'filter', text: 'Filter', value: 'filter' },
+  { key: 'advanced', icon: 'rocket', text: 'Advanced Filter', value: 'advanced' },
```

Comment:
Give it the same icon as the normal filter.

Reply:
And name it accordingly ... `advanced` could be anything.

Reply:
Update icon and naming